SupersonicSound/LowLevel/LowLevelSystem.cs

```diff
@@ -221,11 +221,19 @@ void IPreInitilizeLowLevelSystem.SetCallback(Action<LowLevelSystem, SystemCallba
         #endregion
 
         #region init/close
+        [Obsolete("Not sure why this method is in here, the system should already be initialized when passed in through the constructor")]
         public void Initialize(int maxchannels, LowLevelInitFlags flags)
```

Comment:
Correct, when Studio.System is initialized that automatically initializes the low level system.

I've made an [issue](https://github.com/martindevans/SupersonicSound/issues/7) to address this in the future, but until that's done this isn't obsolete.